contracts/contracts/SimpleWriteAccessController/library.cairo

```diff
@@ -0,0 +1,136 @@
+%lang starknet
+
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+
+from ownable import Ownable_only_owner, Ownable_initializer
+
+@event
+func AddedAccess(user : felt):
+end
+
+@event
+func RemovedAccess(user : felt):
+end
+
+@event
+func CheckAccessEnabled():
+end
+
+@event
+func CheckAccessDisabled():
+end
+
+@storage_var
+func s_check_enabled() -> (checkEnabled : felt):
+end
+
+@storage_var
+func s_access_list(address : felt) -> (bool : felt):
+end
+
+# Adds an address to the access list
+@external
+func add_access{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(user : felt):
+    Ownable_only_owner()
+    _add_access(user)
+    return ()
+end
+
+func _add_access{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(user : felt):
```

Comment:
Why do you have an `_add_access` internal fn but no `_remove_access` internal fn? Let's be consistent here.

Reply:
Removed `_add_access`